server/src/test/resources/config/sidecar_cdc.yaml

```diff
@@ -59,7 +59,7 @@ sidecar:
   cdc:
     segment_hardlink_cache_expiry: 1m # 1 minute
   jmx:
-    max_retries: 42
+    max_retries: 1
```

Comment:
For reviewers, the 42 retries is just too much for testing. The retry blocks the closure of server, leading to the test being stuck for 30 minutes and killed by test runner. 
Changing it to 1 does not alter the tests, but allows test completion way faster (like, in milliseconds).